ironic-source-list

```diff
@@ -1,12 +1,20 @@
 crudini
 gunicorn
 {% if env.IRONIC_SOURCE %}
+    {% if path.isdir('/sources/' + env.IRONIC_SOURCE) %}
```

Comment:
Can we just assume `/sources/ironic` in this case to avoid setting a dummy variable?

Reply:
I'm not sure I follow, IRONIC_SOURCE needs to be set to allow customizing the source from a local dir or a remote location
if we assume /sources/ironic do we want to force the local ironic source dir to always be 'ironic' ?

Reply:
> if we assume /sources/ironic do we want to force the local ironic source dir to always be 'ironic' ?

Yeah, that's what I meant. I don't think we're likely to have anything else :) but it's not a blocking issue.